app/stores/SchoolStore.js

```diff
@@ -104,9 +106,10 @@ function getSchools(schoolIds) {
       return {};
     }
     let addresses = [];
-    const mainBuilding = getMainBuilding(school);
-    if (mainBuilding.building && mainBuilding.building.addresses) {
-      addresses = mainBuilding.building.addresses;
+    const schoolBuilding = getBuildingForYear(school, null);
```

Comment:
You don't need to pass null argument explicitly. Or is it some style convention?